cfgov/unprocessed/apps/owning-a-home/js/explore-rates/rate-checker.js

```diff
@@ -3,26 +3,27 @@ import './tab';
 import 'rangeslider.js';
 import * as params from './params';
 import * as template from './template-loader';
+import Highcharts from 'highcharts';
 import amortize from 'amortize';
 import config from '../../config.json';
 import dropdown from '../dropdown-utils';
 import fetchRates from '../rates';
 import formatUSD from 'format-usd';
-import Highcharts from 'highcharts';
 import highchartsExport from 'highcharts/modules/exporting';
 import isNum from 'is-money-usd';
 import jumbo from 'jumbo-mortgage';
 import median from 'median';
 import unFormatUSD from 'unformat-usd';
 import { applyThemeTo } from './highcharts-theme';
-import { getSelection } from './dom-values';
-import { uniquePrimitives } from '../../../../js/modules/util/array-helpers';
 import {
   calcLoanAmount,
   renderAccessibleData,
   renderDatestamp,
   renderLoanAmount
 } from './util';
+import { getSelection } from './dom-values';
+import { init as tabInit } from './tab.js';
```

Comment:
You don't need the `.js`

Reply:
Actually, can we make this:

`import tab from './tab';`

And then at the initialization line `tab.init();`, so it fits better in the format `object.method()`